[XNNPACK Update] Combined Commit

mcr229 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #139576

No longer have to parse the cmake file because we separated the file definitions from the XNNPACK's bazel build. This let's us just use the same definitions that XNNPACK folk use for their build system!

Differential Revision: [D65392587](https://our.internmc.facebook.com/intern/diff/D65392587/)

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10

mcr229 committed:
[XNNPACK Update] Combined Commit

No longer have to parse the cmake file because we separated the file definitions from the XNNPACK's bazel build. This let's us just use the same definitions that XNNPACK folk use for their build system!

Differential Revision: [D65392587](https://our.internmc.facebook.com/intern/diff/D65392587/)

[ghstack-poisoned]

Skylion007 commented:
Doesn't look like https://github.com/google/XNNPACK/issues/7261 is fixed yet. ;-;

mcr229 committed:
Update on "[XNNPACK Update] Combined Commit"


No longer have to parse the cmake file because we separated the file definitions from the XNNPACK's bazel build. This let's us just use the same definitions that XNNPACK folk use for their build system!

Differential Revision: [D65392587](https://our.internmc.facebook.com/intern/diff/D65392587/)

cc jgong5 mingfeima XiaobingSuper sanchitintel ashokei jingxu10

[ghstack-poisoned]
